src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+const { getErrorWithStatus } = require('./getError');
+
+const getValidId = (id, errorMessage) => {
+  const numberId = Number(id);
+
+  if (isNaN(numberId)) {
+    throw getErrorWithStatus(400, errorMessage);
+  }
+
+  return numberId;
+};
+
+const getValidString = (value, fieldName, expectedType = '') => {
+  if (typeof value !== 'string') {
```

Comment:
Consider adding a check for empty strings if the field is required, to ensure that non-empty strings are provided where necessary.